### PR TITLE
Update project UserGuide: list, edit, find

### LINKED ISSUE #38: Update User Guide
Add features to be implemented in v1.2

### PR DESCRIPTION
**Pull request by WadeForward**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -81,40 +85,39 @@ Examples:
...
+Finds patients whose names contain any of the given keywords.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only the name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
+* patients matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (JinHao-L) [inline docs/UserGuide.md]: Maybe the patients should be caps (for standardisation)

Comment (WadeForward) [reply docs/UserGuide.md]: alright!

**Review by JinHao-L — APPROVED** [review]
LGTM! 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -81,40 +85,39 @@ Examples:
...
 
-Finds persons whose names contain any of the given keywords.
+Finds patients whose names contain any of the given keywords.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only the name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
```
Comment (xz0127) [inline docs/UserGuide.md:119]: Standardise the period at the end of phrases

**Review by xz0127 — COMMENTED** [review]
LGTM except that you may want to add the period to certain phrases. 👍

**Pull request merged**